src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar">
```

Comment:
Due to the task description you have to implement two modifiers for calendar class
![image](https://github.com/mate-academy/layout_calendar/assets/80700247/9467d0af-25ed-4ebf-8253-ad42a6d26b7d)

So no need to add additional classes to list items, the solution should revolve around modifiers